browser/metric_event_mapping.go

```diff
@@ -7,8 +7,9 @@ import (
 )
 
 // mapMetricEvent to the JS module.
-func mapMetricEvent(vu moduleVU, cm *common.MetricEvent) mapping {
-	rt := vu.VU.Runtime()
+func mapMetricEvent(vu moduleVU, event common.PageOnEvent) mapping {
+	rt := vu.Runtime()
```

Comment:
Yeah, some strange linter issue. That's why i used `vu.VU.Runtime()`.

Reply:
I don't want to surrender to the linter's shortcomings :P